packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -158,6 +158,14 @@ export type VideoPlayerEvents = {
    * Handler for an event emitted when the current media source of the player changes.
    */
   sourceChange(newSource: VideoSource, previousSource: VideoSource): void;
+  /**
+   * Handler for an event emitted when the player enters the fullscreen mode.
+   */
+  onFullscreenEnter(): void;
+  /**
+   * Handler for an event emitted when the player exits the fullscreen mode.
+   */
+  onFullscreenExit(): void;
```

Comment:
They are `VideoView` events, not `VideoPlayer` events 😉 You can remove this (and rebuild files).
```suggestion
```

Reply:
Hey @tsapeta , that makes sense. I believe just removing this is not enough and that it needs to be added to the `VideoView.web.tsx`, like so: https://github.com/expo/expo/pull/30922/commits/59fdd579f0a553d23d3c3f3f1d73f87deaa8f28d

I followed @behenate 's PIP implementation as seen here: https://github.com/expo/expo/pull/30524/files#diff-72705cdf87b0dfe8ecd5783a1ffd65f9e6257b81e046c967f4535b75b89c9835R74